src/plugins/filter/utils.ts

```diff
@@ -0,0 +1,167 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import type { FieldArgs } from "grafast";
+import {
+  GraphQLInputType,
+  GraphQLBoolean,
+  GraphQLFloat,
+  GraphQLList,
+  GraphQLString
+} from "graphql";
+
+export enum Operators {
+  EQUAL_TO = "",
+  NOT = "not",
+  IN = "in",
+  NOT_IN = "not_in",
+  GT = "gt",
+  LT = "lt",
+  GTE = "gte",
+  LTE = "lte",
+  CONTAINS = "contains",
+  NOT_CONTAINS = "not_contains",
+  CONTAINS_NOCASE = "contains_nocase",
+  NOT_CONTAINS_NOCASE = "not_contains_nocase",
+  STARTS_WITH = "starts_with",
+  STARTS_WITH_NOCASE = "starts_with_nocase",
+  NOT_STARTS_WITH = "not_starts_with",
+  NOT_STARTS_WITH_NOCASE = "not_starts_with_nocase",
+  ENDS_WITH = "ends_with",
+  ENDS_WITH_NOCASE = "ends_with_nocase",
+  NOT_ENDS_WITH = "not_ends_with",
+  NOT_ENDS_WITH_NOCASE = "not_ends_with_nocase"
+}
+
+// refer "version": "0.0.1-beta.24"  @datallan/pg/dist/codecs.js:686
```

Comment:
Where does this file come from? This reference makes no sense and i cant resolve the source.